base/ltoutput.dtx

```diff
@@ -2224,49 +2283,43 @@
 %    \pkg{footmisc} is called with an option specifying the
 %    footnote placement, i.e., not  in the default case.
 %    In new documents we apply it always.
+%  \changes{v1.4q}{2025/03/20}{Remove a `fil' bottom skip even if we
+%    are in ragged bottom typesetting (footmisc/19)}
 %    \begin{macrocode}
 \def\@outputbox@removebskip{%
 %    \end{macrocode}
-%    We first test if we are in a \cs{raggedbottom} layout. If not we
-%    do nothing, but we don't disable the code because
-%    \cs{raggedbottom} may get used only for some parts of the
-%    document.
-%    \begin{macrocode}
-  \ifx\@textbottom\relax \else
-%    \end{macrocode}
 %    In some special circumstances the \cs{@outputbox} might be
 %    void. As \cs{@outputbox@append} below would change this, we
-%    better handle this case and leave it unchanged in this situation,
-%    because otherwise we can't detect that situation any longer
+%    better handle this case by leaving it unchanged in this situation,
```

Comment:
Here:
we better  ==>  we had better

Better! ?

Reply:
we had better handle ????  I think correct it is just  "we handle this case..."

Reply:
OK: 
   we handle 
is good.